cinn/optim/replace_var_with_expr.cc

```diff
@@ -135,20 +147,23 @@ struct ReplaceVarIndexOfCacheMutator : public ir::IRMutator<> {
 
  private:
   std::map<std::string, ir::Tensor>* global_tensor_map_;
+  std::unordered_set<std::string>* resized_buffer_;
   const Var& var_;
   const Expr& expr_;
   bool blockidx_;
   bool do_replace{false};
+  bool find_replace{false};
```

Comment:
member variable's names should have a `_` suffix.
replace to `do_replace_` and `find_replace_`

Reply:
Done, thanks